lib/main_publiccloud.pm

```diff
@@ -233,10 +239,21 @@ sub load_publiccloud_tests {
             load_maintenance_publiccloud_tests();
         } elsif (get_var('PUBLIC_CLOUD_HIMMELBLAU')) {
             loadtest('publiccloud/himmelblau');
+        } elsif (get_var('PUBLIC_CLOUD_APP_IMG')){
+            my publiccloud_app_img = get_var('PUBLIC_CLOUD_APP_IMG');
```

Comment:
```suggestion
            my publiccloud_app_img = get_required_var('PUBLIC_CLOUD_APP_IMG');
```

I think it would be more clear when variable simply not defined